pkg/protocol/engine/mempool/conflictdag/mocked/conflictdag.go

```diff
@@ -2,7 +2,6 @@
 package mockedconflictdag
```

Comment:
Hide it inside the test package. We have to use the real one in our ledger glue code.